### DISCUSSION TITLE
CrowdStrike User-Agent header added to api_client.py

### DISCUSSION THREAD

**Discussion by PacketJockey** [opencybersecurityalliance/stix-shifter#1062] [Q&A] [answered]
@mdazam1942 I work for CrowdStrike and our team would like to add the user-agent header to the api_client.py.  I have cloned the repo and manually and successfully tested.  It is only a few lines, can we edit the file directly with comments on the changes? 

GitHub newb, just trying to figure out proper etiquette. 

Thanks!

John Smith
Integration Solutions Architect
CrowdStrike

**Comment by mdazam1942** [comment, marked as answer]
@PacketJockey Hi John, You need to create a fork of the stix-shifter repository first. After that you need to create a working branch. 
you can make the changes in your working branch. Once the changes are done and tested, then you need to create a Pull request against the develop branch of the main repository of stix-shifter.

here's a link of how to create fork: Here's a instruction doc of how to create your own fork: https://docs.github.com/en/get-started/quickstart/fork-a-repo

Pull request instruction: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

Let us know if you have more questions.

**Comment by mdazam1942** [comment]
You can also join OCA slack workspace and directly ask questions in #stix-shifter channel. link to join- https://opencybersecurityalliance.org/get-involved/
